src/components/Logs.tsx

```diff
@@ -45,17 +77,22 @@ function Logs({ token, defaultMessage }: Props) {
     );
 
     return (
-        <LazyLog
-            extraLines={1}
-            stream={true}
-            text={logs
+        <Editor
+            height={`${height ?? 200}px`}
```

Comment:
Monaco requires a height so allowing a default but really we should probably not use it. Maybe we can just end up throwing an error if no height is provided.